src/server/multi_svr.c

```diff
@@ -1029,6 +1032,10 @@ update_node_cache(int stream, struct batch_status *bstat)
 
 	init_node_from_bstat(bstat, psvr);
 
+	/* Trigger default scheduler to schedule the job which has failed due
+	to missing vnode cache information */
+	set_scheduler_flag(SCH_SCHEDULE_NEW, dflt_scheduler);
```

Comment:
why only default? I think server should trigger sched cycles for all schedulers that it knows about, otherwise there can be jobs on other scheds which the default sched will ignore

Reply:
We should think about whether we need to support multi-sched in its current form with multi-svr. We can revisit this if we decide to do so.